spec/Section 6 -- Execution.md

```diff
@@ -386,6 +386,8 @@ either resolving to {null} if allowed or further propagated to a parent field.
 If this occurs, any sibling fields which have not yet executed or have not yet
 yielded a value may be cancelled to avoid unnecessary work.
 
+This default error propagation behavior can be modified by using the `@disableErrorPropagation` directive on an operation. If this directive is present, field errors in non-null fields will not propagate to their parent fields. Instead, the field that experienced the error will return `null` (even if it's a non-null field) and the error will be added to the response's `errors` list, but parent fields will continue to be resolved normally. This allows clients to receive partial data with localized errors rather than having errors propagate and potentially nullify large portions of the response.
```

Comment:
We might want to modify the existing text instead of adding a new paragraph. Read in isolation, the paragraph above become misleading.

Maybe just:

```
If during {ExecuteSelectionSet()} a field with a non-null {fieldType} raises a
_field error_ **and the operation does not define `@disableErrorPropagation`** 
then that error must propagate to this entire selection set, ...
```